composer.json

```diff
@@ -64,5 +63,7 @@
         "branch-alias": {
             "dev-master": "2.2.x-dev"
         }
-    }
+    },
+    "prefer-stable": true,
+    "minimum-stability": "dev"
```

Comment:
which dependency needs this? if its a non dev-dependency, the package won't usually install in projects, unless the project changed their minimum-stability...

Reply:
PHPUnit and its dependencies. I didn't need to change this for the cache-plugin because it was already set to dev with prefer stable.

Reply:
I'll see if I can avoid this.